gradio/blocks.py

```diff
@@ -283,7 +283,7 @@ def set_event_trigger(
             "js": js,
             "queue": False if fn is None else queue,
             "api_name": api_name,
-            "scroll_to_output": scroll_to_output,
+            "scroll_to_output": False if utils.get_space() else scroll_to_output,
```

Comment:
You shouldn't change it here -- otherwise even if a developer explicitly sets `scroll_to_output=True` in the event, it won't take effect in Spaces. I would change the the default value in `set_event_trigger()`

Reply:
Disagree, it will break the app on Spaces, so even if it's explicitly allowed we stop it on Spaces. If a user sets it locally and then uploads it to spaces, it shouldn't break the app on spaces.

Reply:
👍